docs/en_US/TrainingService/RemoteMachineMode.rst

```diff
@@ -144,7 +144,7 @@ Files in ``codeDir`` will be uploaded to remote machines automatically. You can
 Configure python environment
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-By default, commands and scripts will be executed in the default environment in remote machine. If there are multiple python virtual environments in your remote machine, and you want to run experiments in a specific environment, then use **preCommand** to specify a python environment on your remote machine. 
+By default, commands and scripts will be executed in the default environment in remote machine. If there are multiple python virtual environments in your remote machine, and you want to run experiments in a specific environment, then use **pythonPath** to specify a python path environment on your remote machine. 
```

Comment:
`python path evnironment` seems strange, `python environment` is enough.

Reply:
Done.